### PR TITLE
`Chore`: Add issue templates

### LINKED ISSUE #287: Create an issue template
We are missing an issue template when creating issues.

### PR DESCRIPTION
**Pull request by DominikRemo**
## Motivation and Context
This project is missing issue templates for bugs/features/custom. Also see issue #287.

## Description
Added issue templates for bug reports and feature requests. Also enabled blank issues, in case an issue does not fit into these categories.

## Steps for Testing
- Please look through the templates and add your feedback

You can see a preview of the issue templates when you go to [bug report](https://github.com/ls1intum/Athena/blob/chore/issue-templates/.github/ISSUE_TEMPLATE/bug-report.yml) and [feature request](https://github.com/ls1intum/Athena/blob/chore/issue-templates/.github/ISSUE_TEMPLATE/feature-request.yml). Note that the description (grey text below the text fields) will appear above the text fields when used as issue templates, not below them.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+name: Bug Report
```
Comment (laadvo) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: Can we use the 🐛 emoji here? I think it looks nicer and it is consistent with Artemis bug reports  that way

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Feature Request
```
Comment (laadvo) [inline .github/ISSUE_TEMPLATE/feature-request.yml]: Here as well, could we use the 🚀 emoji in the name?

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  attributes:
+    value: Thank you for taking the time to submit a feature request! Please describe your request in detail so that we can understand your ideas.
+- type: textarea
+  attributes:
+    label: Is your feature request related to a problem?
+    description: A clear and concise description of what the problem is.
+    placeholder: "For example: I'm always frustrated when [...] or I wish I could [...]."
+- type: textarea
+  attributes:
+    label: Describe the solution you want
```
Comment (laadvo) [inline .github/ISSUE_TEMPLATE/feature-request.yml]:
```suggestion
    label: Describe the solution you'd like
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    description: You can also upload additional material such as mock-ups, diagrams or sketches to support your idea.
```
Comment (laadvo) [inline .github/ISSUE_TEMPLATE/feature-request.yml]:
```suggestion
    description: Feel free to also upload additional material such as mock-ups, diagrams, or sketches to support your idea.
```

**Review by laadvo — COMMENTED** [review]
Looks good, I just have some small suggestions

**Review by laadvo — APPROVED** [review]
thanks for implementing my feedback! LGTM

**Review by maximiliansoelch — APPROVED** [review]

**Pull request merged**